backend/routes/user.js

```diff
@@ -3,6 +3,17 @@ var router = express.Router();
 var User = require("../mongo/User");
 var uuid = require("uuid");
 
+/* GET user information from cookie. */
+router.get("/", async function (req, res, next) {
```

Comment:
I have a get user from cookie method in room that could be abstracted to a seperate file and imported into both. This would reduce duplication.